docs/content/user-guides/docker-compose/basic-example/index.md

```diff
@@ -1,16 +1,15 @@
 ---
 title: "Traefik Docker Documentation"
-description: "This guide covers a Docker Compose file exposing a service using the Docker provider in Traefik Proxy. Read the technical documentation."
+description: "This guide covers a Docker Compose file exposing a service using the Docker provider in Traefik Proxy."
```

Comment:
This was added to improve the SEO, could you check with @tomatokoolaid if this is still ok?

Reply:
Updated after talking with @tomatokoolaid